nms/app/fbcnms-projects/magmalte/app/components/TabUtils.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+import {findIndex} from 'lodash';
+
+// Get current tab position within tabList from url specified
+export function GetCurrentTabPos(url: string, tabItems: Array<string>): number {
```

Comment:
we usually use lower case for util functions, uppercase for classes/components: `getCurrentTabPosition()`

Reply:
similar to how things like `findIndex()` is named :)